src/screens/Server.js

```diff
@@ -73,6 +73,9 @@ class Server extends React.Component {
     hathorLib.wallet.changeServer(newServer);
     const promise = version.checkApiVersion();
     promise.then(() => {
+      // The user might need to change the server before unlocking the wallet when the server he was connected to is down
+      // So after change we unlock the wallet because the PIN was already requested and validated
+      hathorLib.wallet.unlock();
```

Comment:
I don't understand why we need to unlock again. If the PIN has already been validated, wouldn't it be unlocked?

Reply:
This code does not exist anymore. Can you see again, please?